addons/account/models/account_move.py

```diff
@@ -845,6 +853,10 @@ def _compute_invoice_date_due(self):
                 default=False,
             ) or move.invoice_date_due or today
 
+    def _compute_delivery_date(self):
+        for move in self:
+            move.delivery_date = move.delivery_date
```

Comment:
? :smile: 
I don't know which date you would take (`invoice_date`?), but then, maybe you won't need all the changes in the demo for it.